staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -30,6 +30,8 @@ const (
 	NamespaceAll string = ""
 	// NamespaceNodeLease is the namespace where we place node lease objects (used for node heartbeats)
 	NamespaceNodeLease string = "kube-node-lease"
+	// TopologyKeyAny is the service topology key that matches any node
+	TopologyKeyAny string = ""
```

Comment:
If we're still deciding what this should be, I personally like `"*"` more than `""`.

Reply:
How many people agree with change `TopologyKeyAny` from `""` to `"*"` ?

Reply:
I am ok either way. “” is less explicit but “*” might make people expect glob-like behavior which we do not have.

Reply:
I think most people who like "*" more than "", I'm going to change it to "*"

Reply:
Didn't we agree to change this to `"*"` ?

Reply:
yes indeed, and we updated the KEP with that